x-pack/plugins/apm/scripts/kibana-security/setup-custom-kibana-user-role.ts

```diff
@@ -147,13 +147,14 @@ async function init() {
 }
 
 async function isSecurityEnabled() {
-  interface XPackInfo {
-    features: { security?: { allow_rbac: boolean } };
+  try {
+    await callKibana({
+      url: `/api/security/v1/me`,
+    });
+    return true;
+  } catch (err) {
+    return false;
```

Comment:
Shouldn't we only return false here if we have `err.response.status === 404`? Any other response code or type of error would indicate something other than security being disabled.

Reply:
Github ate my comment, but I see what you mean now (I misread it earlier today). I think it should be fine, sounds like an edge case. This is just a check to provide a user-friendly error message.